lib/elixir/lib/code.ex

```diff
@@ -1264,7 +1264,7 @@ defmodule Code do
 
   """
   @spec string_to_quoted_with_comments(List.Chars.t(), keyword) ::
-          {:ok, Macro.t(), map()} | {:error, {location :: keyword, term, term}}
+          {:ok, Macro.t(), list(binary())} | {:error, {location :: keyword, term, term}}
```

Comment:
It should be a list of maps. I think dialyzer can't see inside the list, it only knows it is a list.

Reply:
Updated!